setupContents.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+func externalContents() []byte {
+    return []byte(
+`#!/bin/sh
+set -e
+
+##
+## This code will be run during setup, OUTSIDE the container.
+##
+## Because ` + "`" + `whippet` + "`" + ` running inside the container wouldn't be able to connect
+## to private repositories.
+##
+
+if test -f whippet.json; then
+  whippet deps install
+fi`)
+}
+
+func internalContents() []byte {
+    return []byte(
+`#!/bin/sh
+set -e
+
+##
+## This code will be run during setup, INSIDE the container.
+##
+
+##############
+# Config
+##############
+title="Your site title here"
+theme=your-theme-slug
+plugins="a-space-separated list-of plugins-to-activate"
+content=/usr/src/app/setup/content
+
+wp core !!!INSTALLTYPE!!! --skip-email --admin_user=admin --admin_password=admin --admin_email=admin@localhost.invalid --url=http://localhost --title="$title"
+
+for plugin in $plugins
+do
+  if wp plugin is-installed $plugin
+  then
+    wp plugin activate $plugin !!!ACTIVATIONTYPE!!!
+  else
+    echo "\033[96mWarning:\033[0m Plugin '"$plugin"' could not be found. Have you installed it?"
+  fi
+done
+
+if wp theme is-installed $theme
+then
+  !!!THEMEENABLE!!!
```

Comment:
Check out the examples here: https://godoc.org/text/template#Template

Reply:
Still working on this one.